packages/default-theme/src/components/listing/SwProductListingFilter.vue

```diff
@@ -52,7 +52,7 @@ export default {
       }
     },
     filterCode() {
-      return simplifyString(this.filter.label.toLowerCase(".vue"))
+      return simplifyString(this.filter.label.toLowerCase())
```

Comment:
good point here! missed that in global replace
in 47. line instead of
```js
component: import(`@/components/listing/types/${this.filterCode}`),
```
should probably be
```js
component: import(`@/components/listing/types/${this.filterCode}.vue`),
```

could you add this there?

Reply:
Ok, added it!